tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -108,6 +118,12 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 		]
 	}
 
+	def void updateNullPackageIn(TclModel model, SetupAndCleanupProvider element) {
```

Comment:
If we only have this functionality here, this might cause side-effects. It should probably live in `org.testeditor.tcl.dsl.naming.TclQualifiedNameProvider`. That class is responsible for determining the fully qualified name of an element. There we only need to adapt the qualified name of the `TclModel` (the name of the package if set, if not the inferred package).